### ISSUE TITLE
Accessibility issue with footer cookie preferences

### ISSUE THREAD

**Issue by ingridsamakova** [carbon-design-system/carbon-for-ibm-dotcom#8475]
### Description

Hi there,
Accessibility violation has been found in Footer for Cookie Preferences after running chrome extension IBM Equal Access Accessibility Checker. We are using is in our project IBM Documentation.
Element: <li role="complementary" aria-label="Open Cookie Preferences Modal" consent="0|1|2" data-mode="footer" id="teconsent" data-autoid="dds--privacy-cp" class="bx--legal-nav__list-item truste_caIcon_display">

![Screenshot 2022-03-11 at 23 36 50](https://user-images.githubusercontent.com/101392658/157984869-91ad6630-3f89-4c33-92bd-89216c92af47.png)

### Component(s) impacted

Footer

### Browser

Chrome

### Carbon for IBM.com version

1.30.0

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

IBM Documentation

### Package

@carbon/ibmdotcom-react, @carbon/ibmdotcom-styles

### CodeSandbox example

https://www.ibm.com/standards/carbon/web-components/?path=/story/components-footer--default

### Steps to reproduce the issue (if applicable)

1. Go to Chrome webstore
2. Look for: IBM Equal Access Accessibility Checker extension
3. Install the extension and restart browser
4. open the following link https://www.ibm.com/standards/carbon/react/iframe.html?id=components-masthead--search-open-onload
5. open chrome devtools and choose Accessibility Assessment (next to console/networ/sources/performance)
6. scan the page
7. see violation

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by kennylam** [comment]
@ingridsamakova Using the link you provided, I only see this violation

![image](https://user-images.githubusercontent.com/909118/160722945-fdc26d9b-3192-403b-8ca9-a16ba3f8f5c3.png)

That story you linked only has the `masthead` component, no `footer`. Can you provide a link to your example you were testing?

**Comment by kennylam** [comment]
Fixed in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/8662.
